queries

```diff
@@ -0,0 +1,30 @@
+--This is just a spare file.
```

Comment:
Not a problem at all to have a sandbox file to play around with, but ideally we shouldn't be committing this up to be included in our production code :)

Reply:
Can I keep the files in the git hub for now and delete them at the final stage? don't want to keep tracking them locally.